app/vlstorage/netinsert/netinsert.go

```diff
@@ -325,11 +334,17 @@ func (s *Storage) sendInsertRequestToAnyNode(pendingData *bytesutil.ByteBuffer)
 
 var errTemporarilyDisabled = fmt.Errorf("writing to the node is temporarily disabled")
 
+// RegisterMetricsWriter registers a writer for exposing storage metrics.
+func (s *Storage) RegisterMetricsWriter(w io.Writer) {
```

Comment:
It is better to avoid unused functions and/or stubs, since they make the code less clear.